content/SCALE/SCALETutorials/Apps/UsingTrains.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Managing Catalog Trains"
+description: "Provides basic information on adding or managing application catalogs in TrueNAS SCALE."
+weight: 5
+aliases:
+ - /scale/scaleuireference/apps/usingcatalogs/
+ - /scale/apps/usingcatalogs/
+ - /scale/scaletutorials/apps/usingcatalogs/
+tags:
+- apps
+- customapp
+keywords:
+- nas data storage
+- software storage solutions
+---
+
+TrueNAS SCALE has a pre-built official catalog of available iXsystems-approved applications.
+
+Users can configure add catalog trains if they choose.
+The default train is **stable** which shows the **charts** apps. Other available trains are **enterprise** and **community**.
+
+{{< hint type=note >}}
+TrueNAS uses outbound ports 80/443 to retrieve the TRUENAS catalog.
+{{< /hint >}}
+
+## Managing Catalogs
+Users can manage the catalog trains from the **Settings** option on the **Installed** screen.
+You can also click **Manage Installed Apps** at the top right side of the **Discover** screen to open the **Installed** screen.
+
+{{< trueimage src="/images/SCALE/Apps/AppsDiscoverScreen.png" alt="Applications Discover Screen" id="Applications Discover Screen" >}} new image
+
+Click **Train Settings** on **Settings** menu to open the **Edit Catalog** screen.
```

Comment:
This was a late UI change I was unaware of. This will get fixed in the next sprint under the next ticket when I can review all changes made in the past day or two to update all content.